hikey.mk

```diff
@@ -420,9 +420,33 @@ nvme-cleaner:
 ################################################################################
 # Flash
 ################################################################################
+define flash_help
+	@read -r -p "1. Connect USB OTG cable, the micro USB cable (press any key)" dummy
+	@read -r -p "2. Connect HiKey to power up (press any key)" dummy
+endef
+
+.PHONY: recovery
+recovery:
+	@echo "Enter recovery mode to flash a new bootloader"
+	@echo "Jumper 1-2: Closed (Auto power up)"
+	@echo "       3-4: Closed (Boot Select)"
+	@echo "       5-6: Open   (GPIO3-1)"
```

Comment:
@jforissier , you're right, it doesn't seem like it matter whether 5-6 are open or closed, you end up in recovery mode if 3-4 is closed regardless of the state of 5-6. I wonder why the explicitly tell that this should be open in the getting started. I think I'll remove that since it seems to be irrelevant and I can change according to your suggestion.